common/src/calendar/__tests__/calendar.test.ts

```diff
@@ -658,7 +661,7 @@ describe("isReservationUnitReservable", () => {
         metadataSet: {
           supportedFields: ["name"],
         },
-      } as ReservationUnitByPkType)
+      } as unknown as ReservationUnitByPkType)
```

Comment:
Same as the other PR. Not fan of type casting, common props with spread is better.